setup.py

```diff
@@ -281,6 +281,7 @@ def finalize_options(self):
     provides=['edgedb'],
     zip_safe=False,
     include_package_data=True,
+    package_data={'edgedb': ['py.typed']},
```

Comment:
This is great, thank you for working on this!  The only question I have is why is this line necessary and what does it do.

Reply:
So apparently (I mean, according to PEP 561) the py.typed file allows type checkers to know that this package contains type annotations, and use them when checking dependant packages that import this one.

This basically only has an effect when running mypy against the package, but I noticed that actually most IDEs don’t really care. I’m not sold myself on how intuitive this looks, but it’s in the spec, so 🤷‍♂️.

Interesting related discussion: https://github.com/encode/starlette/pull/455